Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -1961,11 +1962,16 @@ describe(
       return checkDebugColorizeTiles(pointCloudUrl);
     });
 
-    it("debugColorizeTiles for glTF", function () {
+    xit("debugColorizeTiles for glTF", function () {
```

Comment:
You could put a comment here `// see https://github.com/CesiumGS/cesium/issues/10061` to link to the corresponding issue.